src/pyhf/modifiers/shapesys.py

```diff
@@ -27,6 +27,7 @@ def required_parset(cls, sample_data, modifier_data):
             'is_shared': False,
             'inits': (1.0,) * n_parameters,
             'bounds': ((1e-10, 10.0),) * n_parameters,
+            'fixed': False,
```

Comment:
For now, systematic-level application.. but we'd want to change to `(False,)*len(sample_data)` later?